### PR TITLE
PlantingsHelper simplification

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/helpers/plantings_helper.rb [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+    return "0" if planting.finished_at.present? && planting.finished_at_date_passed?
+
+    # days until finished_at
+    return planting.days_until_finished.to_s if planting.finished_at.present?
+
+    return "unknown" if planting.planted_at.nil? || planting.days_before_maturity.nil?
+
+    return "0" if planting.days_until_mature <= 0
+
+    planting.days_until_mature.to_s
```
Comment (Br3nda) [inline app/helpers/plantings_helper.rb]: i think i can make this much clearer....  more changes coming

File: app/models/planting.rb [hunk 2]
```diff
@@ -112,6 +112,21 @@ def planted?(current_date = Date.current)
     planted_at.present? && current_date.to_date >= planted_at
   end
 
+  def finished_at_date_passed?
```
Comment (pozorvlak) [inline app/models/planting.rb]: This method doesn't seem to be used anywhere.

File: app/models/planting.rb [hunk 3]
```diff
@@ -112,6 +112,21 @@ def planted?(current_date = Date.current)
...
+    finished_at.present? && (finished_at - Date.current).to_i <= 0
+  end
+
+  def days_until_finished
+    return 0 if finished?
```
Comment (pozorvlak) [inline app/models/planting.rb]: Should this be `if finished_at_date_passed?`?

File: app/models/planting.rb (hunk 2 above)
Comment (Br3nda) [inline app/models/planting.rb]: You're right, i factored it out in the end. I will remove

File: app/models/planting.rb (hunk 3 above)
Comment (Br3nda) [inline app/models/planting.rb]: No, it's a boolean column on that table of the database. 
```
    t.boolean  "finished",             default: false
```
So, this is checking, is that set to true -- which seems to be yet another way a planting record can be finished.

File: app/models/planting.rb (hunk 2 above)
Comment (Br3nda) [inline app/models/planting.rb]: gone

File: app/models/planting.rb (hunk 3 above)
Comment (pozorvlak) [inline app/models/planting.rb]: Oh dear, that's not very DRY :-(

**Pull request merged**